src/react-apps/applications/shared/src/components/AltinnIconButton.tsx

```diff
@@ -0,0 +1,63 @@
+import { Button, createMuiTheme } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import classNames from 'classnames';
+import * as React from 'react';
+import altinnTheme from '../theme/altinnStudioTheme';
+
+export interface IAltinnIconButtonCompontentProvidedProps {
+  classes: any;
+  onclickFunction?: any;
+  className?: string;
+  iconClass: string;
+  btnText: string;
+}
+
+export interface IAltinnIconButtonComponentProps extends IAltinnIconButtonCompontentProvidedProps {
```

Comment:
Unecessary interface, since it doesn't expand anything.

Reply:
removed